subxt/src/config/signed_extensions.rs

```diff
@@ -258,22 +243,11 @@ impl<T: Config> ChargeAssetTxPayment<T> {
 }
 
 /// Parameters to configure the [`ChargeAssetTxPayment`] signed extension.
-#[derive(Debug)]
```

Comment:
why?

Reply:
it would never have worked anyway afaik, because `T: Config` doesn't impl `Debug`. Maybe I was a bit overzealous in removing a bunch of derives for these things though; I did initially replace them all with working ones and then backed out of it!